### ISSUE TITLE
Follow up of #15949 - Tor state is `Disconnected` in New Private Window with Tor tab

### ISSUE THREAD

**Issue by GeetaSarvadnya** [brave/brave-browser#17381]
## Description 

Tor state is `Disconnected` in New Private Window with Tor tab

## Steps to Reproduce

   1. new profile on `nightly`
   2. open `File` | `Private Window with Tor`
   3. scroll down to the bottom of the tab
   4. look at `Tor Status`

## Actual result:

![image](https://user-images.githubusercontent.com/38657976/128508078-34b45be4-54bf-4019-b191-89e998aa08c9.png)

## Expected result:
Should say `Initializing %` until it's ready, at which point it'll read `Connected` under `Tor Status`

## Reproduces how often: 

Easy

## Brave version (brave://version info)

Brave | 1.28.103 Chromium: 92.0.4515.131 (Official Build) (64-bit)
-- | --
Revision | 6b8d6c56ce21e38a72f7c4becb5abc1fa5134f29-refs/branch-heads/4515@{#1933}
OS | Windows 10 OS Version 2009 (Build 19043.1110)

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards? NA
- Is the issue reproducible on the latest version of Chrome? NA

## Miscellaneous Information:

cc: @brave/legacy_qa @darkdh @rebron

**Comment by stephendonner** [comment]
I couldn't reproduce this with `1.27.111`, but can with `1.28.103`

1.27.111 | 1.28.103
----------|----------
![1-27-11-tor](https://user-images.githubusercontent.com/387249/128540566-ba8125a7-f3fd-42c3-aec0-6833efb21b0f.png) | ![1-28-103-tor](https://user-images.githubusercontent.com/387249/128540569-83edc43f-f2e1-4baa-bf42-b320eadf2d73.png)

**Comment by GeetaSarvadnya** [comment]
@stephendonner Issue is easily reproducible for me in `Windows 10 x64 - 1.27.111`
![Tor status disconnected issue](https://user-images.githubusercontent.com/38657976/128542012-ab511963-3117-40a9-9dc4-3191ee1ada20.gif)

**Comment by kjozwiak** [comment]
I can't seem to reproduce the above using `1.28.103 Chromium: 92.0.4515.131` but seems like @stephendonner, @GeetaSarvadnya and @Awnage can still reproduce as per https://github.com/brave/brave-browser/issues/15949#issuecomment-893118454.

Left a comment via https://github.com/brave/brave-browser/issues/15949#issuecomment-893118454 as well.

**Comment by kjozwiak** [comment]
> @stephendonner Issue is easily reproducible for me in `Windows 10 x64 - 1.27.111`
> ![Tor status disconnected issue](https://user-images.githubusercontent.com/38657976/128542012-ab511963-3117-40a9-9dc4-3191ee1ada20.gif)

@GeetaSarvadnya re: the above GIF, it looks like the state started as `Disconnected` but it started initializing. Have you run into a state where it appears as `Disconnected` and never ends up initializing/connecting?

**Comment by Awnage** [comment]
Is there some way of providing debug logs or something that would help? I have tried with Stable, Beta, and Nightly without success. I turned off all my firewalls, just in case, still no luck.

**Comment by kjozwiak** [comment]
> Is there some way of providing debug logs or something that would help? I have tried with Stable, Beta, and Nightly without success. I turned off all my firewalls, just in case, still no luck.

Which issue are you running into? That sometimes it starts in a `Disconnected` mode or that you can never connnect?

**Comment by Awnage** [comment]
@kjozwiak Starts in a `Disconnected` state

**Comment by stephendonner** [comment]
> Is there some way of providing debug logs or something that would help? I have tried with Stable, Beta, and Nightly without success. I turned off all my firewalls, just in case, still no luck.

yes: check out `brave://tor-internals/` - the `Logs` tab there 👍

**Comment by Awnage** [comment]
Still the blank as before: https://github.com/brave/brave-browser/issues/15949#issuecomment-880013913
```
Tor Version:
Tor Process Id: -1
Tor Proxy URI:
Tor Connection Status: Disconnected
Tor Initialization Progress:
```

**Comment by Awnage** [comment]
Still happening with `Version 1.29.81 Chromium: 93.0.4577.82 (Official Build) (64-bit)`

**Comment by geoff-scott** [comment]
*BUMP*

 I am experiencing the same issue myself, I was doing a Google search to see if I could find a solution. Seems like I am not the only one.
 
Version details are as follows:

 [Version 1.35.101 Chromium: 98.0.4758.87 (Official Build) (64-bit)
 Windows 10 Pro

**Comment by geoff-scott** [comment]
> Still happening with `Version 1.29.81 Chromium: 93.0.4577.82 (Official Build) (64-bit)`

BUMP

I am experiencing the same issue myself, I was doing a Google search to see if I could find a solution. Seems like I am not the only one.

Version details are as follows:

[Version 1.35.101 Chromium: 98.0.4758.87 (Official Build) (64-bit)
Windows 10 Pro
